hammett/core/hiders.py

```diff
@@ -75,7 +76,10 @@ async def run(self: 'Self', update: 'Update', context: 'CallbackContext') -> boo
                 msg = f"The hider '{hider}' is unregistered"
                 raise HiderIsUnregistered(msg) from exc
 
-            if await hider_handler(update, context):
+            if asyncio.iscoroutinefunction(hider_handler):
+                if await hider_handler(update, context):
+                    return True
+            elif hider_handler(update, context):
```

Comment:
We allow define sync funcion as hider_handler? It nullifies asynchrony

Reply:
In simple cases such as `return settings.IS_ADMIN` you don't have async hider handlers.